packages/amplify/amplify_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 1.3.1
```

Comment:
callout: output of `aft version-bump` was "NEXT" here and did not actually change the version on amplify_flutter. I thought that would be odd, so I manually changed this to 1.3.1 and manually changed the version in amplify_flutter's pubspec. I'm open to other ideas for resolving this if not desired as we usually keep this changelog as a top-level changelog akin to release notes.

Reply:
Shoot. That's a bug.